manifests/profile/subscriber.pp

```diff
@@ -9,10 +9,9 @@
 # This profile is the base class for
 # profiles subscribing to riemann::profile::proxy
 #
-class riemann::profile::subscriber
-{
-  include ::riemann
-  include ::riemann::profile
-  include ::riemann::subscription
+class riemann::profile::subscriber {
```

Comment:
Thou mitely script shall  not school  an corky wizard how to perform magic!

Reply:
Yes yes, we can `--no-manifest_whitespace_opening_brace_before`